README.md

```diff
@@ -125,35 +132,24 @@ files.
         ),
     );
 
-#### Using parent aliases
-
-A little known feature of drush site aliases is that you can specify a comma-separated
-list of site aliases whose properties will be inherited by this alias. To do so, add a
-'parent' key to the alias array:
+#### Parent alias
+A little known feature of drush site aliases is that you can specify a comma-separated list of site aliases whose properties will be inherited by this alias. To do so, add a 'parent' key to the alias array:
 
     'parent' => '@etc, @grandparent, @parent',
 
-This can be handy when managing multiple configurations of sites in your multisite
-setup. You can define the characteristics that are common to a given configuration
-in the parent alias, and then in the child site, just provide those details that
-are needed for that site.
+This can be handy when managing multiple configurations of sites in your multisite setup. You can define the characteristics that are common to a given configuration in the parent alias, and then in the child site, just provide those details that are needed for that site.
 
-#### Options
-
-You can use the following flags:
 
-    --simulate
+## Contributing
 
-flag to see what the commands will do to your system. When using simulate there
-will be no changes to your system.
+Contact the authors if you want to submit pull requests to us.
 
-    -f
+## Authors
 
-Force the script to continue when errors are encountered. E.g.:
-You have tried to install a site but your failed halfway due to permissions.
-You can then change the permissions on your server, and force the script to execute.
+* **Chris Warren** - *Initial version of Sitespinner in PHP*
```

Comment:
This is not correct. Sitespinner was originally written as bash scripts by Andy Cavenaugh, then at Common Media. I rewrote it as a drush command. Chris did not do any coding, as I recall, but of course he developed the concept and oversaw development of both versions.